docs/cookbook.rst

```diff
@@ -2,6 +2,19 @@
 Recipes
 =======
 
+This plugin is an adapter for the microphone of ScreenPy's Narrator which sends logs to Allure.
+
+This `example <https://github.com/ScreenPyHQ/screenpy_examples/tree/trunk/screenpy_selenium/github>`__ makes use of this plugin.
+
+Rationale
+=========
+
+Without this plugin when you generate a test report using allure it will display as follow
+
+.. image:: ./allure_without_plugin.png
+  :width: 768
+  :alt: Screenshot without plugin has not the steps
```

Comment:
I don't think this section is necessary—if someone's arrived at this documentation, they are probably here because Allure doesn't show the steps, so they probably don't need to see the "before" shot.